hawk/app/assets/stylesheets/authentication/_content.scss

```diff
@@ -32,13 +37,17 @@
     margin: 50px 20px 55px 20px;
 
     a {
-      color: #00b268;
+      color: #059AAA;
       text-decoration: none;
 
       &:hover {
-        color: #00b268;
+        color: #059AAA;
         text-decoration: none;
       }
+
+      &.upstream {
+      color: #059AAA;
```

Comment:
Color literals like `#059AAA` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#059AAA` should be written as `#059aaa`<br>Line should be indented 8 spaces, but was indented 6 spaces